featuretools/tests/utils_tests/test_entity_utils.py

```diff
@@ -17,19 +16,14 @@
 
 
 @pytest.fixture
-def pd_mock_customer_es():
-    return ft.demo.load_mock_customer(return_entityset=True, random_seed=0)
-
-
-@pytest.fixture
-def dask_mock_customer_es(pd_mock_customer_es):
-    dask_es = copy.deepcopy(pd_mock_customer_es)
+def dask_mock_customer_es(pd_mock_customer):
+    dask_es = copy.deepcopy(pd_mock_customer)
```

Comment:
We can delete this and use the `dd_mock_customer` fixture from `conftest.py` instead.